src/clarity-angular/datagrid/datagrid-row.ts

```diff
@@ -3,15 +3,19 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import {Component, Input, Output, EventEmitter} from "@angular/core";
+import {Component, Input, Output, EventEmitter, ContentChildren, QueryList} from "@angular/core";
 import {Selection} from "./providers/selection";
+import {DatagridActionOverflow} from "./datagrid-action-overflow";
 
 @Component({
     selector: "clr-dg-row",
     template: `
         <clr-dg-cell *ngIf="selection.selectable" class="datagrid-select">
             <clr-checkbox [ngModel]="selected" (ngModelChange)="toggle($event)"></clr-checkbox>
         </clr-dg-cell>
+        <clr-dg-cell *ngIf="actionableRows.length > 0" class="datagrid-single-select">
```

Comment:
Nitpick: do you mind renaming this `datagrid-single-select` class to `datagrid-row-actions`?
We might have actual single selection patterns coming in later, and I want to avoid potential conflicts.